content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_two_steps.rst

```diff
@@ -4,185 +4,229 @@ Process receipts and deliveries in two steps
 
-Click :guilabel:`Validate` to complete the transfer and move the product from :guilabel:`WH/Input`
-to :guilabel:`WH/Stock`.
+Once ready, click :guilabel:`Validate` to complete the transfer and move the product from
+:guilabel:`WH/Input` to :guilabel:`WH/Stock`.
 
-Once the transfer is validated, the product enters inventory and is available for customer
+Once the transfer is validated, the products enter inventory, and are available for customer
 deliveries or manufacturing orders.
 
 .. image:: receipts_delivery_two_steps/receipts-delivery-two-steps-internal-transfer.png
    :align: center
-   :alt: Validate the internal transfer to move the item to stock.
+   :alt: Internal transfer form for products ordered from vendor.
 
-Process a delivery order in two steps (pick + ship)
-===================================================
+Process delivery order in two steps (pick + ship)
+=================================================
 
-Create a sales order
---------------------
+When products are delivered in two steps, they move from warehouse stock to an output location, then
+from the output location to a customer location in the database upon validation of a picking order,
+and a subsequent delivery order (DO).
 
-In the :menuselection:`Sales` application, create a new quote by clicking :guilabel:`New`. Select
-(or create) a :guilabel:`Customer`, add a storable :guilabel:`Product` to the order lines, and then
-click :guilabel:`Confirm`.
+Create sales order
+------------------
 
-After confirming the :abbr:`SO (Sales Order)`, a :guilabel:`Delivery` smart button will appear in
-the top, above the :abbr:`SO (Sales Order)` form. Click the :guilabel:`Delivery` smart button to
-reveal the associated receipt.
+To create a |SO|, navigate to the :menuselection:`Sales app`, and click :guilabel:`New`. This
+opens a blank sales quotation form.
 
-.. image:: receipts_delivery_two_steps/receipts-delivery-two-steps-delivery-button.png
+Add a customer in the :guilabel:`Customer` field. Then, fill out the various fields on the sales
+quotation form, as necessary.
+
+.. image:: receipts_delivery_two_steps/receipts-delivery-two-steps-new-sales-order.png
    :align: center
-   :alt: Delivery smart button on sales order form.
+   :alt: Filled out new sales order form.
 
-.. tip::
-   Sales order receipts can also be found in the :menuselection:`Inventory` application. In the
-   :guilabel:`Overview` dashboard, click the :guilabel:`# To Process` smart button in the
-   :guilabel:`Pick` Kanban card.
+Under the :guilabel:`Product` tab, click :guilabel:`Add a product`, and select a product to add to
+the sales order quotation.
+
+Once ready, click :guilabel:`Confirm`. This moves the quotation to the :guilabel:`Sales Order`
+stage.
+
+Once the |SO| is confirmed, a :guilabel:`Delivery` smart button appears at the top of the form.
+Clicking the smart button opens the warehouse delivery (WH/OUT) form.
```

Comment:
```suggestion
Once the |SO| is confirmed, a :guilabel:`Delivery` smart button appears at the top of the form.
Clicking the smart button opens the warehouse delivery (WH/OUT) form.
```
I'm seeing two delivery actions in runbot, your image shows one, why might this be?

![2024-05-03_10-10](https://github.com/odoo/documentation/assets/91896312/a062d556-b516-4861-b137-bfb778d24abe)

Reply:
In 17.2 the triggers for subsequent operations are different - populate with 1 instead of 2 and located elsewhere in the app